apps/condo/domains/ticket/access/Ticket.js

```diff
@@ -88,6 +92,12 @@ async function canManageTickets ({ authentication: { item: user }, operation, it
             return ticket.client === user.id
         }
     } else {
+        // NOTE: service user with any fields pass
```

Comment:
probably you need to check it for service user too.

Reply:
fixed